Language/Functions/Math/abs.adoc

```diff
@@ -72,5 +72,14 @@ a++;  // keep other math outside the function
 [float]
 === See also
 
+[role="definition"]
```

Comment:
```suggestion
[role="language"]
```

According to the standardized reference sample:
https://github.com/arduino/reference-en/blob/master/AsciiDoc_sample/Reference_Terms/AsciiDoc_Template-Single_Entity.adoc?plain=1#L111

Please make the same change on the other pages as well.